src/Extensions/ElementalAreasExtension.php

```diff
@@ -168,9 +178,12 @@ public function updateCMSFields(FieldList $fields)
             return;
         }
 
-        // add an empty holder for content as some module explicitly use insert
-        // after content.
-        $fields->replaceField('Content', new LiteralField('Content', ''));
+        // add an empty holder for content as some module explicitly use insert after content
+        $globalReplace = !Config::inst()->get(self::class, 'keep_content_fields');
```

Comment:
I'd not use the `!` here, but in the if statement below. It makes more sense to assign a variable to its actual value and check in the if instead of the other way around

Reply:
I did this due to the name of the variable and the name of the config settings basically being opposites in intention. I made it this way to be easier to process (in a human brain).

I termed the config value `keep` in order to reflect the default better - the absence of the config value (i.e. `null`) should be in line with the default behaviour I thought. This makes sense on individual models (page types) where there won't be a setting, unless a developer specifically sets one. I then transferred this to the global value as well (which was previously 'replace') in order to reduce cognitive load with the two config values (for global and individual models) being opposite to each other. I thought it easier to align one's brain around the logic if both were termed with the same intention.

I found the variable name (which is opposite behaviour to the config value) to make more sense in the if statement, in a form of self documenting code. I found it difficult to name things in a way that made sense when it came to the if statement referring to 'keep' settings guarding the _remove_ statement, coupled trying to keep the boolean logic of the if statement in line with sensible human language (and not being too verbose in itself).